src/simplewallet/simplewallet.cpp

```diff
@@ -6714,7 +6645,7 @@ bool simple_wallet::ons_update_mapping(std::vector<std::string> args)
     }
 
     auto& enc_hex = response[0].encrypted_value;
-    if (!oxenmq::is_hex(enc_hex) || enc_hex.size() % 2 != 0 || enc_hex.size() > 2*ons::mapping_value::BUFFER_SIZE)
+    if (!oxenc::is_hex(enc_hex) || enc_hex.size() % 2 != 0 || enc_hex.size() > 2*ons::mapping_value::BUFFER_SIZE)
```

Comment:
isn't the size mod 0 check redundant with the one preceding?

Reply:
Yes, but in some early versions of oxenmq `is_hex` it wasn't which it why it was here.  It is now, though, so I'll update.